src/validation/Validator.ts

```diff
@@ -332,6 +341,30 @@ export class Validator {
         return value instanceof Boolean || typeof value === "boolean";
     }
 
+
+/**
+* Checks if a given value is a latitude.
+*/
+    isLatLong(value: any): boolean {
+
+        return validator.isLatLong(value);
+    }
+
+    isLatitude(value: any): boolean {
+        const lat = /^\(?[+-]?(90(\.0+)?|[1-8]?\d(\.\d+)?)$/;
+   
+        return lat.test(value);
```

Comment:
As I said before use `validator.js` not regex extracted from `validator.js` library. In this case:
```ts
return (typeof value === "number" || this.isString(value)) && this.isLatLong(`${value},0`);
```

Reply:
validatorjs doesnot have a single islatitude and islongitude , so we have to use regex extracted from that

Reply:
Sure thats why we will use ```this.isLatLong(`${value},0`)```.

Reply:
oh okay